backend/tasks/user.py

```diff
@@ -0,0 +1,87 @@
+from flask_restful import Resource, reqparse
+from flask import request
+from psycopg2.extras import RealDictCursor
+from pymysql.cursors import DictCursor
+import os
+from flask import make_response
+import json
+import logging
+from flask_login import login_required
+logger = logging.getLogger(__name__)
+
+
+class UserTask(Resource):
+
+    @login_required
+    def get(self, user):
+        if user is None or len(user) == 0:
+            return make_response(json.dumps({'message': 'Parameter \'user\' is required'}), 400)
+
+        try:
+            with request.db.cursor(cursor_factory=RealDictCursor) if os.environ.get('IS_LOCAL') else request.db.cursor(DictCursor) as cur:
```

Comment:
This logic should be hidden somewhere in the database layer, perhaps we need a wrapper around the cursor with a method `dict_cursor`. We can follow up on it later.

Reply:
Agreed. Once again seemed out of scope for this feature. But should be implemented in the future.

Reply:
Definitely agreed. I'm sure there's going to be more use cases in the future.